data.json

```diff
@@ -14289,7 +14318,10 @@
     },
     "natural-selection-2": {
         "linux": {
-            "root": "steam_icon_4920"
+            "root": "steam_icon_4920",
+            "symlinks": [
+                "spaghetti-armageddon"
```

Comment:
I see why the symlink, but as we've learned with #5585 this isn't great practice for non-generic icons just because they look similar. Possibly make a copy of that but with colours from the game? Might be a good opportunity to neaten up that icon too.